### PR TITLE
CLI & Docker

### PR DESCRIPTION
**Pull request by TheSeems**
Add basic CLI functionality & Dockerfile

### CODE REVIEW

File: simdev/main.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import logging
```
Comment (EgorBu) [inline simdev/main.py]: why don't you keep `__main__.py`? It allows to skip `main` during the call of the module `python3 -m simdev.main`? If you keep rename this file to `__main__.py` - you can call it `python3 -m simdev`

**Review by EgorBu — CHANGES_REQUESTED** [review]
Let's rename `main.py` to `__main__.py` and I will merge it

File: simdev/main.py (hunk 1 above)
Comment (TheSeems) [reply simdev/main.py]: Renamed!

**Review by EgorBu — APPROVED** [review]
Good!

**Pull request merged**